twake/frontend/src/app/components/Leftbar/Channel/Channel.scss

```diff
@@ -378,3 +378,12 @@
     }
   }
 }
+
+.ant-avatar-group .ant-avatar:not(:first-child) {
+  margin-left: -12px !important;
+}
+
+.ant-badge-dot {
+  width: 8px !important;
+  height: 8px !important;
+}
```

Comment:
This style modification seems global to me and related to the Twake theme (Am I right ?) can you then move it to theme.less ?